nemo/collections/asr/metrics/wer.py

```diff
@@ -1199,34 +247,40 @@ def on_validation_epoch_end(self):
 
     def __init__(
         self,
-        decoding: CTCDecoding,
+        decoding: Union[AbstractCTCDecoding, AbstractRNNTDecoding],
         use_cer=False,
         log_prediction=True,
-        fold_consecutive=True,
+        batch_dim_index=0,
```

Comment:
Why remove fold consecutive ? Its needed for SSL.

Reply:
I thought that was for the CTC decoding only (where we never call it). Lemme throw back in.

Reply:
Oh, that's why, we do call it on SLU. mb